assignments/array-methods.js

```diff
@@ -56,21 +56,32 @@ const runners = [{"id":1,"first_name":"Charmain","last_name":"Seiler","email":"c
 // ==== Challenge 1: Use .forEach() ====
 // The event director needs both the first and last names of each runner for their running bibs.  Combine both the first and last names into a new array called fullName. 
 let fullName = [];
+runners.forEach((item) => {
```

Comment:
With ES6 Arrow Function Syntax, its optional to include parenthesis around 1 parameter only.. if 2 or more, they are required.. your code COULD look like this:
runners.forEach( item => {